ksp_plugin/interface_flight_plan.cpp

```diff
@@ -255,14 +255,14 @@ XYZ __cdecl principia__FlightPlanGetGuidance(Plugin const* const plugin,
     result = plugin->renderer().BarycentricToWorld(
                  plugin->PlanetariumRotation())(manœuvre.InertialDirection());
   } else {
-    result = plugin->CurrentTime() < manœuvre.initial_time()
-                 ? plugin->renderer().BarycentricToWorld(
-                       plugin->PlanetariumRotation())(
-                       manœuvre.FrenetFrame()(manœuvre.direction()))
-                 : plugin->renderer().FrenetToWorld(
-                       *plugin->GetVessel(vessel_guid),
-                       *manœuvre.frame(),
-                       plugin->PlanetariumRotation())(manœuvre.direction());
+    result = (plugin->CurrentTime() < manœuvre.initial_time()
+                  ? plugin->renderer().BarycentricToWorld(
+                        plugin->PlanetariumRotation()) *
+                        manœuvre.FrenetFrame()
+                  : plugin->renderer().FrenetToWorld(
+                        *plugin->GetVessel(vessel_guid),
+                        *manœuvre.frame(),
+                        plugin->PlanetariumRotation()))(manœuvre.direction());
```

Comment:
Note that the two branches would be more similar (and the code more transparent) if we had a `FrenetToBarycentric` instead of a `FrenetToWorld` because the barycentric-to-world conversion could be factored out.

Reply:
Good point.

This would also be far more readable if Manœuvre::FrenetFrame were called Manœuvre::InitialFrenetFrame.

Let’s do these cleanups in a subsequent pull request.